helm-charts/whitesource-renovate/templates/NOTES.txt

```diff
@@ -1,9 +1,9 @@
-{{- if not .Values.renovate.acceptWhiteSourceTos }}
-ERROR: You must accept the WhiteSource Terms of Service in order to run WhiteSource Renovate.
+{{- if not .Values.renovate.acceptMendTos }}
```

Comment:
noting this and resolving it with my next commit -- I assume the app itself would freak out and that's beyond the scope of this PR (or maybe not, you tell me)